src/sentry/config.js

```diff
@@ -0,0 +1,14 @@
+import * as Sentry from 'sentry-expo';
+import {SENTRY_DSN} from '@env'
+
+var sentryConfig = {
+    dsn: SENTRY_DSN,
+    enableInExpoDevelopment: true,
+    debug: true, // Sentry will try to print out useful debugging information if something goes wrong with sending an event. Set this to `false` in production.
+}
+
+function initSentry(){
+    Sentry.init(sentryConfig)
+}
+
```

Comment:
I would add a function here like `logError(Error err)` and `log(String message)` so that the front end does not have to use the sentry api directly. This is a good practice since we might be changing the logger out later on in the future or upgrading to a new version. The front end wont notice if we change it and that's the point.